spec/std/socket_spec.cr

```diff
@@ -217,6 +242,21 @@ describe UNIXServer do
     end
   end
 
+  it "does not delete any file on close for abstract server" do
```

Comment:
Oh, I misread the test, it's actually checking if it doesn't try to delete a file. Let's keep it.